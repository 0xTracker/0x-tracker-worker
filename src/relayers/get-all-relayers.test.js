const getAllRelayers = require('./get-all-relayers');

beforeEach(() => {
  jest.resetAllMocks();
});

it('should return all relayers', () => {
  const relayers = getAllRelayers();

  expect(relayers).toMatchSnapshot();
});
