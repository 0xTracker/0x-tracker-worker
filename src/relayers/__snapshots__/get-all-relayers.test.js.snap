// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return all relayers 1`] = `
Object {
  "bambooRelay": Object {
    "feeRecipients": Array [
      "0x5dd835a893734b8d556eccf87800b76dda5aedc5",
      "0xc898fbee1cc94c0ff077faa5449915a506eff384",
    ],
    "id": "bambooRelay",
    "imageUrl": "https://0xtracker.com/assets/logos/bamboo-relay.png",
    "lookupId": 1,
    "name": "Bamboo Relay",
    "slug": "bamboo-relay",
    "url": "https://bamboorelay.com/",
  },
  "bit2MeDex": Object {
    "feeRecipients": Array [
      "0x1b38d2613698488cbac0e7d21abb2fdbbe1d6704",
    ],
    "id": "bit2MeDex",
    "imageUrl": "https://0xtracker.com/assets/logos/bit2me-dex.png",
    "lookupId": 27,
    "name": "Bit2Me DEX",
    "slug": "bit2me-dex",
    "url": "https://bit2me.com/dex/",
  },
  "boxSwap": Object {
    "feeRecipients": Array [
      "0x66a836664adc7c525c0cc4527dee8619d4faf669",
    ],
    "id": "boxSwap",
    "imageUrl": "https://0xtracker.com/assets/logos/box-swap.png",
    "lookupId": 14,
    "name": "Boxswap",
    "slug": "boxswap",
    "url": "https://boxswap.io/",
  },
  "ddex": Object {
    "id": "ddex",
    "imageUrl": "https://0xtracker.com/assets/logos/ddex.png",
    "lookupId": 2,
    "name": "DDEX",
    "orderMatcher": true,
    "slug": "ddex",
    "takerAddresses": Array [
      "0xe269e891a2ec8585a378882ffa531141205e92e9",
    ],
    "url": "https://ddex.io",
  },
  "deversifi": Object {
    "feeRecipients": Array [
      "0x61b9898c9b60a159fc91ae8026563cd226b7a0c1",
      "0xaf8ae6955d07776ab690e565ba6fbc79b8de3a5d",
    ],
    "id": "deversifi",
    "imageUrl": "https://0xtracker.com/assets/logos/deversifi.png",
    "lookupId": 21,
    "name": "DeversiFi",
    "slug": "deversifi",
    "url": "https://www.deversifi.com/",
  },
  "emoon": Object {
    "feeRecipients": Array [
      "0xb0d7398d779ee9ffc727d2d5b045a5b441da8233",
    ],
    "id": "emoon",
    "imageUrl": "https://0xtracker.com/assets/logos/emoon.png",
    "lookupId": 19,
    "name": "Emoon",
    "slug": "emoon",
    "url": "https://emoon.io",
  },
  "ercDex": Object {
    "feeRecipients": Array [
      "fred",
      "flintstone",
    ],
    "id": "ercDex",
    "imageUrl": "https://0xtracker.com/assets/logos/erc-dex.png",
    "lookupId": 3,
    "name": "ERC dEX",
    "slug": "erc-dex",
    "url": "https://ercdex.com",
  },
  "etherBlockchain": Object {
    "feeRecipients": Array [
      "0x2a5f5f36c20d7e56358db78bbfac0bace25c1198",
    ],
    "id": "etherBlockchain",
    "imageUrl": "https://0xtracker.com/assets/logos/ether-blockchain.png",
    "lookupId": 22,
    "name": "EtherBlockchain.io",
    "slug": "ether-blockchain",
    "url": "https://www.etherblockchain.io/",
  },
  "fordex": Object {
    "id": "fordex",
    "imageUrl": "https://0xtracker.com/assets/logos/fordex.png",
    "lookupId": 20,
    "name": "Fordex",
    "orderMatcher": true,
    "slug": "fordex",
    "takerAddresses": Array [
      "0x853da5cecc1ea601ab978c2001565a0377a7dca6",
    ],
    "url": "https://www.fordex.co",
  },
  "godsUnchained": Object {
    "id": "godsUnchained",
    "imageUrl": "https://0xtracker.com/assets/logos/gods-unchained.png",
    "lookupId": 29,
    "name": "Gods Unchained",
    "senderAddresses": Array [
      "0xb04239b53806ab31141e6cd47c63fb3480cac908",
    ],
    "slug": "gods-unchained",
    "url": "https://godsunchained.com/market",
  },
  "guDecks": Object {
    "feeRecipients": Array [
      "0x013ec57d1237e7727f818b1a35e3506f754304e4",
      "0xd9c3ed92520a2f3076a9bd5ea9d4a2c2a0cf6457",
    ],
    "id": "guDecks",
    "imageUrl": "https://0xtracker.com/assets/logos/gudecks.png",
    "lookupId": 16,
    "name": "GUDecks",
    "slug": "gu-decks",
    "url": "https://gudecks.com/",
  },
  "idtExchange": Object {
    "feeRecipients": Array [
      "0xeb71bad396acaa128aeadbc7dbd59ca32263de01",
    ],
    "id": "idtExchange",
    "lookupId": 4,
    "name": "IDT Exchange",
    "slug": "idt-exchange",
  },
  "instex": Object {
    "feeRecipients": Array [
      "0x3151e8ab9bfe754ada6682b9d75906e06a18a741",
    ],
    "id": "instex",
    "imageUrl": "https://0xtracker.com/assets/logos/instex.png",
    "lookupId": 17,
    "name": "Instex",
    "slug": "instex",
    "url": "https://instex.io",
  },
  "ledgerDex": Object {
    "feeRecipients": Array [
      "0x4524baa98f9a3b9dec57caae7633936ef96bd708",
    ],
    "id": "ledgerDex",
    "imageUrl": "https://0xtracker.com/assets/logos/ledger-dex.png",
    "lookupId": 12,
    "name": "LedgerDex",
    "slug": "ledger-dex",
    "url": "https://ledgerdex.com",
  },
  "marketProtocolExchange": Object {
    "feeRecipients": Array [
      "0x8752d14a086cee9b8c108611ba9aefe04042c9f9",
    ],
    "id": "marketProtocolExchange",
    "imageUrl": "https://0xtracker.com/assets/logos/market-protocol-exchange.png",
    "lookupId": 23,
    "name": "MARKET Protocol Exchange",
    "slug": "market-protocol-exchange",
    "url": "https://mpexchange.io/",
  },
  "merchcoinsExchange": Object {
    "feeRecipients": Array [
      "0x584860fe460a16cd11e6fdebb2e77146d1835be8",
    ],
    "id": "merchcoinsExchange",
    "imageUrl": "https://0xtracker.com/assets/logos/merchcoins.png",
    "lookupId": 25,
    "name": "Merchcoins Exchange",
    "slug": "merchcoins-exchange",
    "url": "https://merchcoins.com/",
  },
  "oc2Dex": Object {
    "feeRecipients": Array [
      "0x1af29de80f743be444aa195e2cae38ee9bc74c4c",
      "0x879047421625b87b5895ef1ac0d9dbd241805aa3",
    ],
    "id": "oc2Dex",
    "imageUrl": "https://0xtracker.com/assets/logos/oc2-dex.png",
    "lookupId": 26,
    "name": "Oc2 DEX",
    "slug": "oc2-dex",
    "url": "https://oc2realm.com/",
  },
  "odee": Object {
    "feeRecipients": Array [
      "0x382310cbb159b64c2e7c5675d110202701a436dd",
    ],
    "id": "odee",
    "imageUrl": "https://0xtracker.com/assets/logos/odee.png",
    "lookupId": 33,
    "name": "Odee",
    "slug": "odee",
    "url": "https://odee.com",
  },
  "oneInchExchange": Object {
    "feeRecipients": Array [
      "0x4d37f28d2db99e8d35a6c725a5f1749a085850a3",
    ],
    "id": "oneInchExchange",
    "imageUrl": "https://0xtracker.com/assets/logos/1inch-exchange.png",
    "lookupId": 32,
    "name": "1inch.exchange",
    "slug": "1inch-exchange",
    "url": "https://1inch.exchange",
  },
  "openRelay": Object {
    "feeRecipients": Array [
      "0xc22d5b2951db72b44cfb8089bb8cd374a3c354ea",
      "0x89db81c2dc4adaf10a93705b69289d479d576635",
    ],
    "id": "openRelay",
    "imageUrl": "https://0xtracker.com/assets/logos/open-relay.png",
    "lookupId": 5,
    "name": "OpenRelay",
    "slug": "open-relay",
    "url": "https://openrelay.xyz",
  },
  "paradex": Object {
    "id": "paradex",
    "imageUrl": "https://0xtracker.com/assets/logos/paradex.png",
    "lookupId": 6,
    "name": "Paradex",
    "orderMatcher": true,
    "slug": "paradex",
    "takerAddresses": Array [
      "0x4969358e80cdc3d74477d7447bffa3b2e2acbe92",
      "0xd2045edc40199019e221d71c0913343f7908d0d5",
    ],
    "url": "https://paradex.io",
  },
  "radarRelay": Object {
    "feeRecipients": Array [
      "0xa258b39954cef5cb142fd567a46cddb31a670124",
    ],
    "id": "radarRelay",
    "imageUrl": "https://0xtracker.com/assets/logos/radar-relay.png",
    "lookupId": 7,
    "name": "Radar Relay",
    "slug": "radar-relay",
    "url": "https://radarrelay.com",
  },
  "sharkRelay": Object {
    "feeRecipients": Array [
      "0x55890b06f0877a01bb5349d93b202961f8e27a9b",
    ],
    "id": "sharkRelay",
    "imageUrl": "https://0xtracker.com/assets/logos/shark-relay.png",
    "lookupId": 8,
    "name": "Shark Relay",
    "slug": "shark-relay",
    "url": "https://sharkrelay.com",
  },
  "starBit": Object {
    "feeRecipients": Array [
      "0x8124071f810d533ff63de61d0c98db99eeb99d64",
      "0xc370d2a5920344aa6b7d8d11250e3e861434cbdd",
    ],
    "id": "starBit",
    "imageUrl": "https://0xtracker.com/assets/logos/starbit.png",
    "lookupId": 9,
    "name": "STAR BIT",
    "orderMatcher": true,
    "slug": "star-bit",
    "takerAddresses": Array [
      "0x0681e844593a051e2882ec897ecd5444efe19ff2",
    ],
    "url": "https://www.starbitex.com",
  },
  "theOcean": Object {
    "feeRecipients": Array [
      "0x7219612be7036d1bfa933e16ca1246008f38c5fe",
    ],
    "id": "theOcean",
    "imageUrl": "https://0xtracker.com/assets/logos/the-ocean.png",
    "lookupId": 13,
    "name": "The Ocean",
    "orderMatcher": true,
    "slug": "the-ocean",
    "url": "https://theocean.trade/",
  },
  "tokenJar": Object {
    "feeRecipients": Array [
      "0x5e150a33ffa97a8d22f59c77ae5487b089ef62e9",
    ],
    "id": "tokenJar",
    "imageUrl": "https://0xtracker.com/assets/logos/token-jar.png",
    "lookupId": 10,
    "name": "Token Jar",
    "slug": "token-jar",
    "url": "https://tokenjar.io/",
  },
  "tokenTrove": Object {
    "feeRecipients": Array [
      "0x0d056bb17ad4df5593b93a1efc29cb35ba4aa38d",
    ],
    "id": "tokenTrove",
    "imageUrl": "https://0xtracker.com/assets/logos/token-trove.png",
    "lookupId": 28,
    "name": "TokenTrove",
    "slug": "token-trove",
    "url": "https://tokentrove.com",
  },
  "tokenlon": Object {
    "feeRecipients": Array [
      "0x6f7ae872e995f98fcd2a7d3ba17b7ddfb884305f",
      "0xb9e29984fe50602e7a619662ebed4f90d93824c7",
    ],
    "id": "tokenlon",
    "imageUrl": "https://0xtracker.com/assets/logos/tokenlon.png",
    "lookupId": 11,
    "name": "Tokenlon",
    "slug": "tokenlon",
    "url": "https://tokenlon.im/instant",
  },
  "tokenmom": Object {
    "id": "tokenmom",
    "imageUrl": "https://0xtracker.com/assets/logos/tokenmom.png",
    "lookupId": 15,
    "name": "Tokenmom",
    "orderMatcher": true,
    "slug": "tokenmom",
    "takerAddresses": Array [
      "0x4a821aa1affbf7ee89a245bf750d1d7374e77409",
    ],
    "url": "https://tokenmom.com",
  },
  "udex": Object {
    "feeRecipients": Array [
      "0xa5004c8b2d64ad08a80d33ad000820d63aa2ccc9",
    ],
    "id": "udex",
    "imageUrl": "https://0xtracker.com/assets/logos/udex.png",
    "lookupId": 30,
    "name": "UDEX",
    "slug": "udex",
    "url": "https://fridayte.ch/",
  },
  "veil": Object {
    "id": "veil",
    "imageUrl": "https://0xtracker.com/assets/logos/veil.png",
    "lookupId": 18,
    "name": "Veil",
    "slug": "veil",
    "takerAddresses": Array [
      "0xdf1bc6498338135de5ffdbcb98817d81e2665912",
    ],
    "url": "https://veil.co/",
  },
  "veriDex": Object {
    "feeRecipients": Array [
      "0x5265bde27f57e738be6c1f6ab3544e82cdc92a8f",
    ],
    "id": "veriDex",
    "imageUrl": "https://0xtracker.com/assets/logos/veridex.png",
    "lookupId": 24,
    "name": "VeriDEX",
    "slug": "veridex",
    "url": "https://dex.verisafe.io/",
  },
  "zeroExApi": Object {
    "feeRecipients": Array [
      "0x1000000000000000000000000000000000000011",
    ],
    "id": "zeroExApi",
    "imageUrl": "https://0xtracker.com/assets/logos/0x.png",
    "lookupId": 31,
    "name": "0x API",
    "slug": "0x-api",
    "url": "https://0x.org/api",
  },
}
`;
